infrastructure/variables.tf

```diff
@@ -100,3 +100,11 @@ variable "tenant_id" {}
 variable "jenkins_AAD_objectId" {
   type        = "string"
 }
+
+variable "postgresql_user" {
+  default = "notification"
+}
+
+variable "database_name" {
+  default = "notification"
+}
```

Comment:
Please add postgres username and password for all other environments using vault.